packages/server-ct/src/project-ct.js

```diff
@@ -106,6 +109,18 @@ class Project {
       projectRoot: this.projectRoot,
       configFile: settings.pathToConfigFile(this.projectRoot, options),
     })
+    .then((modifiedCfg) => {
+      if (modifiedCfg.browsers.length) {
+        return Promise.resolve(modifiedCfg)
+      }
+
+      // add chrome as a default browser if none has been specified
+      return browsers.ensureAndGetByNameOrPath(DEFAULT_BROWSER_NAME).then((browser) => {
+        modifiedCfg.browsers = [browser]
+
+        return modifiedCfg
+      })
```

Comment:
Why wouldn't you always call `ensureAndGetByNameOrPath`? I think it makes sense to validate they have Chrome installed.

```js
return browsers.ensureAndGetByNameOrPath(DEFAULT_BROWSER_NAME).then((browser) => {
  if (browser) modifiedCfg.browsers = [browser]

  return modifiedCfg
})
```

Reply:
I guess I don't understand what we're trying to avoid calling and what the functionality we're sidestepping is.

Reply:
@JessicaSachs `browsers` is expecting an array of objects, but we only have the name of the browser. 
This function (not very well named) resolves the first browser that corresponds to this name and returns it as an object.

I took the code written by @brian-mann and copied it since I knew it worked.
I think the function here should be called `browsers.resolveBrowserFromName(DEFAULT_BROWSER_NAME)` instead of this.

Do you prefer, that I add comments or that I do an implementation of this `resolveBrowserFromName` function?